extensions/ql-vscode/src/remote-queries/view/CodePaths.tsx

```diff
@@ -94,25 +94,22 @@ const Menu = ({
   codeFlows: CodeFlow[],
   setSelectedCodeFlow: (value: React.SetStateAction<CodeFlow>) => void
 }) => {
-  return <ActionMenu>
-    <ActionMenu.Anchor>
-      <Button variant="invisible" sx={{ fontWeight: 'normal', color: 'var(--vscode-editor-foreground);', padding: 0 }} >
-        {getCodeFlowName(codeFlows[0])}
-        <TriangleDownIcon size={16} />
-      </Button>
-    </ActionMenu.Anchor>
-    <ActionMenu.Overlay sx={{ backgroundColor: 'var(--vscode-editor-background)' }}>
-      <ActionList>
-        {codeFlows.map((codeFlow, index) =>
-          <ActionList.Item
-            key={`codeflow-${index}'`}
-            onSelect={(e: React.MouseEvent) => { setSelectedCodeFlow(codeFlow); }}>
-            {getCodeFlowName(codeFlow)}
-          </ActionList.Item>
-        )}
-      </ActionList>
-    </ActionMenu.Overlay>
-  </ActionMenu>;
+  return <VSCodeDropdown
+    onChange={(event: any) => {
+      const selectedOption = event.target;
+      const selectedIndex = selectedOption.value;
+      setSelectedCodeFlow(codeFlows[selectedIndex]);
```

Comment:
I'm aware this is could be written more concisely as 
```js
setSelectedCodeFlow(codeFlows[event.target.value]);
```

But I found the verbose way easier to understand 😂 (React states are confusing 🙃 )

Reply:
What you chose is exactly what I would have written!